src/sos/tasks.py

```diff
@@ -1685,26 +1685,26 @@ def row(th=None, td=None):
     elif verbosity == 0:
         print("\n".join(obtained_status))
     elif verbosity == 1:
-        for s, (t, d) in zip(obtained_status, all_tasks):
+        for s, (t, dr) in zip(obtained_status, all_tasks):
             print(f"{t}\t{s}")
     elif verbosity == 2:
         tsize = 20
-        for s, (t, d) in zip(obtained_status, all_tasks):
+        for s, (t, dr) in zip(obtained_status, all_tasks):
```

Comment:
This one?

Reply:
I concluded that d was a typo here as well since it was an unused variable. Should I revert it back to the original format?

Reply:
I just submitted a commit that replaces the variable dr with an underscore for lines 1688, 1692 and 1699.